Tinylog/View Controllers/Tasks/TasksViewController.swift

```diff
@@ -708,65 +634,65 @@ final class TasksViewController: CoreDataTableViewController,
-                    let taskItem: TLITask = task as! TLITask
-                    let displayLongText: NSString = NSString(format: "- %@\n", taskItem.displayLongText!)
-                    output = output.appending(displayLongText as String) as NSString
+        guard let list = list,
+            let title = list.title else {
+            return
+        }
+        
+        do {
+            let fetchRequest: NSFetchRequest<NSFetchRequestResult> = NSFetchRequest(entityName: "Task")
+            let positionDescriptor  = NSSortDescriptor(key: "position", ascending: false)
+            let displayLongTextDescriptor  = NSSortDescriptor(key: "displayLongText", ascending: true)
+            fetchRequest.sortDescriptors = [positionDescriptor, displayLongTextDescriptor]
+            fetchRequest.predicate = NSPredicate(format: "list = %@", list)
+            fetchRequest.fetchBatchSize = 20
+            let tasks: NSArray = try managedObjectContext.fetch(fetchRequest) as NSArray
+
+            var outputString: String = ""
+            outputString = outputString.appending("\(title)\n")
+            
+            tasks.forEach { task in
+                guard let task = task as? TLITask,
+                    let displayLongText = task.displayLongText else {
+                    return
                 }
-
-                let activityViewController: UIActivityViewController = UIActivityViewController(
-                    activityItems: [output], applicationActivities: nil)
-                activityViewController.excludedActivityTypes = [
-                    UIActivity.ActivityType.postToTwitter,
-                    UIActivity.ActivityType.postToFacebook,
-                    UIActivity.ActivityType.postToWeibo,
-                    UIActivity.ActivityType.copyToPasteboard,
-                    UIActivity.ActivityType.assignToContact,
-                    UIActivity.ActivityType.saveToCameraRoll,
-                    UIActivity.ActivityType.addToReadingList,
-                    UIActivity.ActivityType.postToFlickr,
-                    UIActivity.ActivityType.postToVimeo,
-                    UIActivity.ActivityType.postToTencentWeibo
-                ]
-
-                activityViewController.modalPresentationStyle = UIModalPresentationStyle.popover
-                activityViewController.popoverPresentationController?.sourceRect = sender.bounds
-                activityViewController.popoverPresentationController?.sourceView = sender
-                activityViewController.popoverPresentationController?.permittedArrowDirections
-                    = UIPopoverArrowDirection.any
-
-                self.navigationController?.present(
-                    activityViewController, animated: true, completion: nil)
-            } catch let error as NSError {
-                fatalError(error.localizedDescription)
+                outputString = outputString.appending("- \(displayLongText)\n")
             }
+            
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/sger/Tinylog/blob/61b2e19801c35844b857fed4761ebd1cb7f36195/Tinylog/View Controllers/Tasks/TasksViewController.swift#L670"></span>Lines should not have trailing whitespace.<br />
<code>trailing_whitespace</code> <code>TasksViewController.swift:670</code></td>
    </tr>
  </tbody>
</table>